netuitive/conf/collectors/DNSLookupCheckCollector.conf

```diff
@@ -1,5 +1,3 @@
 enabled = False
 ttl = 150
-
-#Replace www.google.com and www.yahoo.com with the DNS names you want to check
-dnsAddressList = www.google.com, www.yahoo.com
+dnsAddressList = host01.example.com, host02.example.com
```

Comment:
Is a trailing comma needed? We've seen that in a few other places.

Could you remove the subdomain from one example to make it clear that's _not_ necessary?

Reply:
To my knowledge, a trailing comma isn't needed. The proposed changes are intended to be cosmetic, not functional. Should it be determined comma is required, a separate issue can be created and PR submitted.

I went ahead and removed the subdomain from the second entry.